scripts/run_fvt_docker

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export FABRIC_FVT_HOST=fabric-fvt
+export MYSQL_DOCKER_CERTS=.mysql_data/mysql
+export MYSQL_DOCKER_HOST=MySQL_Server_5.7.27_Auto_Generated_Server_Certificate
+export MYSQL_VER=5.7
+export PG_VER=9.5
+export POSTGRES_DOCKER_HOST=postgres
+
+function killAndRemove() {
+  if [[ "$(docker ps -a -q -f name="${1}")" ]]; then
+    docker rm -f "${1}"
+  fi
+}
+
+function cleanup() {
+  printf "\nCleaning up FVT Docker artifacts...\n"
+  killAndRemove ${MYSQL_DOCKER_HOST}
+  killAndRemove ${POSTGRES_DOCKER_HOST}
+  killAndRemove ${FABRIC_FVT_HOST}
+  if [[ "$(docker network ls -q -f name=fvt)" ]]; then
+    docker network rm fvtnet
+  fi
+  rm -rf .mysql_data
+}
+
+cleanup
+docker network create fvtnet
+docker run --network fvtnet -d --name ${POSTGRES_DOCKER_HOST} -e POSTGRES_PASSWORD=postgres postgres:${PG_VER}
+docker run --network fvtnet -d --name ${MYSQL_DOCKER_HOST} -v $(pwd)/${MYSQL_DOCKER_CERTS}:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=mysql mysql:${MYSQL_VER}
+docker run --network fvtnet --name ${FABRIC_FVT_HOST} -v $(pwd):/opt/gopath/src/github.com/hyperledger/fabric-ca hyperledger/fabric-ca-fvt
+
+trap cleanup EXIT
```

Comment:
This will eventually become the standard file targeted by the fvt-tests make target, for users who run locally. We will have a second target `fvt-tests-ci` that will implement service containers and not use this file at all